Dockerfile

```diff
@@ -9,13 +9,16 @@
 #
 # Description: Docker container image recipe
 
-FROM debian:jessie as builder
+FROM debian:jessie-slim as builder
 
 LABEL maintainer="Fossology <fossology@fossology.org>"
 
 WORKDIR /fossology
 
-RUN DEBIAN_FRONTEND=noninteractive apt-get update \
+# Fix for Postgres and other packages in slim variant
+RUN mkdir /usr/share/man/man1 \
+ && mkdir /usr/share/man/man7 \
```

Comment:
Creating these two directories is not necessary.

Reply:
Done